### ISSUE TITLE
a11y: Clicks and long-clicks don't produce appropriate sound on Android

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#14187]
Clicking and long-clicking an element on Android iwith Talkback does not produce the appropriate feedback sounds and announcements of what got activated.

We probably needs to dispatch AccessibilityEvents TYPE_VIEW_CLICKED and TYPE_VIEW_LONG_CLICKED in responses to clicks and long-clicks to fix this.

**Comment by goderbauer** [comment]
Also missing: haptic feedback (probably produced by same events).

**Comment by meggamind** [comment]
@goderbauer  @Hixie  I can work on this

**Comment by meggamind** [comment]
@goderbauer I'm working on this. I'm new to flutter and contributing to it, so taking time, should be done in a couple of days

**Comment by meggamind** [comment]
@goderbauer I need help reproducing this, I double clicked a raised button in a11y mode and it make the appropriate sound. perhaps I'm doing something wrong. Is there a slack/IRL channel for flutter community ?

**Comment by goderbauer** [comment]
I just did:

```
$ flutter create a11y_test
$ cd a11y_test
$ flutter run
```

and then on my Android device (running Oreo) I focused with talkback on the floating action button in the bottom right. When I now double tap the screen I don't hear audio feedback that the tap occured.

**Comment by meggamind** [comment]
I have recorded a video with a11y_test app giving audio feedback on click (on pixel 2 Oreo)
https://streamable.com/1bnou  
Am I missing something ? Are you referring to the talk back feedback saying 'button clicked' ?

**Comment by goderbauer** [comment]
Hopefully we can remove the work around we put in place for seelction controlls when we fix this: https://github.com/flutter/engine/blob/master/shell/platform/android/io/flutter/view/AccessibilityBridge.java#L566

**Comment by jonahwilliams** [comment]
Fixed by  https://github.com/flutter/flutter/pull/16945 and https://github.com/flutter/engine/pull/5081
